cfme/containers/provider/openshift.py

```diff
@@ -30,7 +30,7 @@ def create(self, validate_credentials=True, **kwargs):
         # probably a js wait issue, not reproducible manually
         super(OpenshiftProvider, self).create(validate_credentials=validate_credentials, **kwargs)
 
-    def _href(self):
+    def href(self):
```

Comment:
I didn't event tried (It'll work) but it just a convention:
http://docs.quantifiedcode.com/python-code-patterns/correctness/accessing_a_protected_member_from_outside_the_class.html

I think this method should be public and not protected.
If you insist - I can keep it protected and call it outside - but this is not recommended...